remote/src/main/resources/reference.conf

```diff
@@ -184,6 +184,20 @@ pekko {
     # is 'off'. Set this to 'off' to suppress these.
     warn-unsafe-watch-outside-cluster = on
 
+    # When receiving requests from other remote actors, what are the valid
+    # prefix's to check against. Useful for when dealing with rolling cluster
+    # migrations with compatible systems such as Lightbend's Akka.
+    accept-protocol-names = ["pekko", "akka"]
```

Comment:
could we mention that these 2 properties were only added in v1.0.2?

Reply:
Do you have an example elsewhere in the docs on the phrasing/formatting of this?

Reply:
We have comment text in reference.conf files in this repo for things like deprecations (and the version that the deprecation happened in). I thought it would be uncontroversial to have text that says config was only added in v1.0.2.

Reply:
I would have thought that something like `# This config was added in v1.0.2.` would be ok.

Reply:
It's a matter of style not controversy, all good I'll find an example

Reply:
So I just search both `1.` and `2.` in only `*.conf` files and it appears that historically Akka never documents when config values are added, likely because its documented elsewhere (both in actual documentation and migration notes). 

Basically the version that a config is added is implicitly derived from the fact that the documentation will refer to it (and if the config isn't there then its not going to be in the documentation).

Reply:
I think it is better to doc this in the reference.conf even if we also document it in the paradox docs.

Reply:
Ill have a think about it but I am not seeing the value here given that the config literally won't exist in the docs in earlier versions and if was an issue we would have at least a single example of it.